glide.go

```diff
@@ -404,6 +404,10 @@ Example:
 					Name:  "use-gopath",
 					Usage: "Copy dependencies from the GOPATH if they exist there.",
 				},
+				cli.BoolFlag{
+					Name:  "strip",
```

Comment:
my only nitpick would be that the name could be more obvious.  --rmvcs or --strip-vcs or something would be clearer, but I'm not looking a gift horse in the mouth.